src/client/app/inert.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  type App,
+  computed,
+  inject,
+  reactive,
+  type UnwrapNestedRefs
+} from 'vue'
+
+const inertSymbol = Symbol()
+const inertStateSymbol = Symbol()
+
+export interface Inert {
+  isSidebarOpen: boolean
+  isScreenOpen: boolean
+  isSidebarEnabled: boolean
+  onAfterRouteChanged: () => void
+}
+
+export interface InertState {
+  inertSkipLink: boolean
+  inertNav: boolean
+  inertLocalNav: boolean
+  inertSidebar: boolean
+  inertContent: boolean
+  inertFooter: boolean
+}
+
+export function useInert() {
+  return inject<UnwrapNestedRefs<Inert>>(inertSymbol)
+}
+
+export function useInertState() {
+  return inject<UnwrapNestedRefs<InertState>>(inertStateSymbol)
+}
+
+export function provideInert(app: App) {
+  const inert = reactive({
+    isSidebarOpen: false,
+    isScreenOpen: false,
+    isSidebarEnabled: false,
+    onAfterRouteChanged() {
```

Comment:
use `onAfterRouteChanged` on `nav.ts`

Reply:
@brc-dd I'm going to expose directly the logic since we are exposing both composables

Reply:
we cannot expose it, it is being used `@click="closeScreen"`